### DISCUSSION TITLE
Automatically send invite who star App-Choreography/Get-An-Invite repo

### DISCUSSION THREAD

**Discussion by vedantmgoyal9** [App-Choreography/OurContributors#38] [Team Posts]
Not sure if you (@CodingSpecies) have already thought about it, but it will save time so we can focus on other things such as organizing contests, etc...

**Comment by MetaMars0114** [comment]
Ok

**Comment by trinib** [comment]
@vedantmgoyal2009 Good idea. 

@CodingSpecies Also posting on reddit with a large [github community](https://www.reddit.com/search/?q=github&type=sr) at the right day and time can get repo ALOT of attention if you are lucky with time of posting(weekends early morning 6 AM and 9 AM). Most of the time repositories do not reach to people unless if you are some famous coder or YouTuber 

ps: do not post all over reddit else they will remove post or might ban from community. Other users will share it across other communities. Good luck👍

**Comment by attractiveDev** [comment]
Hello, everyone!.

**Comment by NS007-dev** [comment]
Hi all!

@vedantmgoyal2009 I had thought about automating invites and I actually tried to install a bot to do so, but it didn't seem to work. If any of you have any ideas on how we can do so, please share!

**Comment by NS007-dev** [comment]
> @vedantmgoyal2009 Good idea.
> 
> @CodingSpecies Also posting on reddit with a large github community at the right day and time can get repo ALOT of attention if you are lucky with time of posting(weekends early morning 6 AM and 9 AM). Most of the time repositories do not reach to people unless if you are some famous coder or YouTuber
> 
> ps: do not post all over reddit else they will remove post or might ban from community. Other users will share it across other communities. Good luck👍

Thanks for the advice @trinib ! I will look into posting on Reddit!

**Comment by NS007-dev** [comment]
@vedantmgoyal2009 I have made a token, and an org secret. Thank you for the useful guidance.

So in your PR, once I merge it what do I edit to suit the changes I just made? :
![image](https://user-images.githubusercontent.com/70807500/198298084-e1a38482-8356-4399-8713-4224c1f166d1.png)

**Comment by NS007-dev** [comment]
Right, ok @vedantmgoyal2009 !
Thank you!

**Comment by NS007-dev** [comment]
PR merged 🎉
Let's see the results the next time someone opens a join org issue!

**Comment by trinib** [comment]
> @vedantmgoyal2009 Good idea.
> 
>     @CodingSpecies Also posting on reddit with a large github community at the right day and time can get repo ALOT of attention if you are lucky with time of posting(weekends early morning 6 AM and 9 AM). Most of the time repositories do not reach to people unless if you are some famous coder or YouTuber
> 
>     ps: do not post all over reddit else they will remove post or might ban from community. Other users will share it across other communities. Good luck👍
> 
> Thanks for the advice @trinib ! I will look into posting on Reddit!

ps you can post your content on different reddit communties BUT have title and description differently from each other but still point to your github content 😎

**Comment by trinib** [comment]
@CodingSpecies i got lucky with this repo https://github.com/trinib/Linux-Bash-Commands in reddit [programming](https://www.reddit.com/r/programming/)  community ..  I just did it for fun and did not realize it would of got so many stars lol 1500 in two days .. i posted on Saturday at 8:30am
